plugins/destination/postgresql/client/write.go

```diff
@@ -45,8 +37,7 @@ func (c *Client) Write(ctx context.Context, tables schema.Tables, res <-chan *pl
 					// not recoverable error
 					return fmt.Errorf("failed to execute batch: %w", err)
 				}
-				atomic.AddUint64(&c.metrics.Errors, 1)
-				c.logger.Error().Err(pgErr).Str("table", pgErr.TableName).Msg("failed to execute batch with pgerror")
+				return fmt.Errorf("failed to execute batch with pgerror on table %s: %w", pgErr.TableName, err)
```

Comment:
Since this stops the sync (correct me if I'm wrong), maybe add a call to action?
`Try skipping this table via 'skip_tables: ["<table_name>"]' and re-running the sync command`

Reply:
I think no need, better we get the report and fix it asap. This is basically equivalent of panic, it shouldn't happen mostly as validation should happen in our type system.